partner_credit_limit_warning/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+# -*- encoding: utf-8 -*-
+# © 2014 Elico corp(www.elico-corp.com)
+# Licence AGPL-3.0 or later(http://www.gnu.org/licenses/agpl.html)
+
+
+{
+    'name': 'Check Partner credit limit',
+    'version': '7.0.1.0.0',
+    'category': 'Sales',
+    'description': """
+        Check the credit limit of the customer and:
+        - Adds one warning when clicking on SO confirm button
+        - block the delivery order process button.
+        Allows a manager to authorize the delivery 
+    """,
+    'author': 'Elico Corp',
+    'website': 'http://www.openerp.net.cn',
```

Comment:
this is the old URL please make sure to check in all modules and change to www.elico-corp.com
Thanks